### ISSUE TITLE
FR: consider adding `default-audit-criteria` to config.toml

### ISSUE THREAD

**Issue by gburgessiv** [mozilla/cargo-vet#408]
At the moment, ChromeOS is auditing crates in two stages:

- 'First pass' audits are for `"safe-to-run"` and a local `"crypto-safe"` key
- As a later pass, we expect folks will audit for `"safe-to-deploy"`

While we're in a purely first-pass stage, we're having to add manual policy blocks to `config.toml` for each top-level crate:

```
[policy.foo]
criteria = ["safe-to-run", "crypto-safe"]
dev-criteria = ["safe-to-run", "crypto-safe"]
```

It would be nice if we could instead specify a top-level `default-audit-criteria = ["safe-to-run", "crypto-safe"]` (and might scare people less, since they're less likely to see a "700KLOC of backlog" message if they add a new top-level crate without adding the requisite `policy` line :) )

cc @bholley since we talked about this offline

**Comment by bholley** [comment]
To add a bit here: our thinking in the design here was that policy inheritance would generally avert significant duplication, but ChromeOS has lots of top-level crates. So it would be nice to configure the policy table defaults somehow, not totally sure yet what the cleanest syntax for that would be.

**Comment by mystor** [comment]
We could consider adding a `[default-policy]` section or something like that, which would have a similar syntax to `[policy]` (though with only support for the `criteria` and `dev-criteria` keys). This would then apply to all root crates which don't specify their own policy rather than the normal hard-coded defaults?

So you'd open your `config.toml` file with something like:

```toml
[default-policy]
criteria = ["safe-to-run", "crypto-safe"]
dev-criteria = ["safe-to-run", "crypto-safe"]

[policy.foo]
# ... whatever ...
```

**Comment by bholley** [comment]
Yeah that sounds like the right approach. I think we'd need to restrict it to a subset of fields — dependency-criteria and audit-as-crates-io wouldn't make sense in this context — but that should be straightforward.
